wsclient/src/test/java/org/hpccsystems/ws/client/platform/test/PlatformTester.java

```diff
@@ -283,68 +278,16 @@ else if (currentParam.matches(WSSQLPORTPATTERN))
-                if (actualclusternames != null)
-                {
-                    for (int j = 0; j < actualclusternames.length; j++)
-                    {
-                        System.out.println("\t" + actualclusternames[j]);
-                    }
-                }
-            }
-
-            //pick one of the valid clusters
-            connector.sprayFlatHPCCFile("people-small", "mythor::persons",Persons.recLen, clusters.get(0) /*myroxie, mythor, etc*/, true);
-
-
-            String outputFlatfileContentsEcl = Persons.elcLayout +
-                    "hpccpersonrecords := DATASET('~mythor::persons', Layout_Persons, FLAT);" +
-                   "OUTPUT(hpccpersonrecords,, 'processed::persons',OVERWRITE);";
-
-            WorkunitInfo wu=new WorkunitInfo();
-            wu.setECL(outputFlatfileContentsEcl);
-            wu.setJobname("mypersonsfile");
-            wu.setCluster(clusterGroups[0]);
-            wu.setResultLimit(10);
-            wu.setMaxMonitorMillis(50000);
-            //this is just one way to submitECL, you can also submit via ecldirect and request the resulting WUID
-            //you can also, submit via WSWorkunits and have more control over the result window you get back.
-            String results = connector.submitECLandGetResults(wu);
-
-            System.out.println("Persons file content(10 recs) output: " + results);
 
-            EclRecordInfo datasetFields = wsDFUClient.getDatasetFields("hthor::processed::persons", null, "," );
-
-            HashMap<String,DFURecordDefInfo> recordsets = datasetFields.getRecordsets();
-            Set<String> keySet = recordsets.keySet();
-            for (String field : keySet)
+            DFUDataColumn[] newgetFileDataColumns = wsDFUClient.getFileMetaData(".::kw_test_sup", null);
+            for (int i = 0; i < newgetFileDataColumns.length; i++)
             {
-                DFURecordDefInfo dfuRecordDefInfo = recordsets.get(field);
-                List<DFUDataColumnInfo> childColumns = dfuRecordDefInfo.getChildColumns();
-                for (DFUDataColumnInfo dfuDataColumnInfo : childColumns)
-                {
-                    System.out.println(dfuDataColumnInfo.getColumnEclType() + " " + dfuDataColumnInfo.getColumnLabel());
-                }
-            }
-
-            List<DFUDataColumnInfo> newgetFileDataColumns = wsDFUClient.getFileMetaDataInfo("hthor::processed::persons", null);
-            for (int i = 0; i < newgetFileDataColumns.size(); i++)
-            {
-                System.out.println("Col name: " + newgetFileDataColumns.get(i).getColumnLabel() + " ecl: " + newgetFileDataColumns.get(i).getColumnEclType() + " col type " + newgetFileDataColumns.get(i).getColumnType());
+                System.out.println("Col name: " + newgetFileDataColumns[i].getColumnLabel() + " ecl: " + newgetFileDataColumns[i].getColumnEclType() + " col type " + newgetFileDataColumns[i].getColumnType());
```

Comment:
The syntax var[index].method() is used elsewhere in our code.  If there were any worries about backwards compatibility with java 7 like we talked about, I would expect us to run into problems everywhere, not just here.

Reply:
ok sounds good. Still concerning that a different syntax was used at some point.